packages/core/src/controllers/LionWebBulkApi.ts

```diff
@@ -6,6 +6,7 @@
 import { Request, Response } from "express"
 import { LionWebJsonChunk, LionWebValidator } from "@lionweb/validation"
 import { LIONWEB_BULKAPI_WORKER } from "./LionWebBulkApiWorker.js"
+import {requestsVerbosity} from "../server.js";
```

Comment:
Stylistic inconsistency: did you run `prettier`?

Reply:
No. How can I do that? If you can tell me, I will run the command, add an entry in package.json and add a reference to it in the README

Reply:
Oh wait: `prettier` is not (yet) configured in this repo. That means you could fix this manually, or choose to forget about it ;)

Reply:
There is a .prettierrc file in the repository, and the is a prettier dependency in the package.json. However, you need to run prettier by hand currently.

Reply:
I ran prettier and that changed 69 files. So I would rather ignore stylistic issues in this file and open a separate PR where I add the task for prettier and I apply formatting to the whole codebase (see https://github.com/LionWeb-io/lionweb-repository/issues/28). Would this work for you?

Reply:
Agreed!